sentry-uitest/sentry-uitest-android-benchmark/benchmark-proguard-rules.pro

```diff
@@ -0,0 +1,35 @@
+# Add project specific ProGuard rules here.
```

Comment:
Maybe we rename the `sentry-uitest` folder to something more generic, since we have benchmarking and UI tests.
Maybe `sentry-android-integration-tests` or something else? since more benchmarking will come and not necessarily mean there's UI in there.